providers/innertube/src/main/kotlin/it/vfsfitvnm/innertube/models/PlayerResponse.kt

```diff
@@ -35,6 +35,18 @@ data class PlayerResponse(
         val highestQualityFormat: AdaptiveFormat?
             get() = adaptiveFormats?.findLast { it.itag == 251 || it.itag == 140 }
 
+        val lowestQualityFormat: AdaptiveFormat?
```

Comment:
I think this should be based on `bitrate` or `averageBitrate`?

Reply:
Bitrate, i guess, based on the format provided by youtube.
[https://gist.github.com/AgentOak/34d47c65b1d28829bb17c24c04a0096f#dash-audio](url)

Thank you for the feedback!